src/main/java/com/sforce/ws/bind/XmlObject.java

```diff
@@ -302,6 +330,42 @@ protected void loadAfterStartTag(XmlInputStream in, TypeMapper typeMapper) throw
         if (textFound) value = typeMapper.deserialize(text.toString(), xmlType);
     }
 
+    private XmlObject extractChildElement(XmlInputStream in, TypeMapper typeMapper) throws ConnectionException {
```

Comment:
Is there an implementation of this logic in TypeMapper? It seems like this should already exist somewhere. This isn't a big deal, but it would be nice.